utils/pdftoppm.cc

```diff
@@ -264,20 +268,25 @@ static GBool parseJpegOptions()
   return gTrue;
 }
 
+static GBool shouldProcessAnnotation(Annot *, void * pflag) {
+  return (*((bool*)pflag)) ? gTrue : gFalse;
+};
```

Comment:
this is a pretty direct port of pdf2htmlEX's flag.

for ref:
https://github.com/coolwanglu/pdf2htmlEX/blob/271bd3aeb25e33ad466a804b53c277d26256a0e1/src/BackgroundRenderer/CairoBackgroundRenderer.cc#L110